atomic_reactor/plugins/pre_pull_base_image.py

```diff
@@ -101,5 +101,11 @@ def run(self):
             self.log.error("giving up trying to pull image")
             raise RuntimeError("too many attempts to pull and tag image")
 
+        # Make sure registry image is tagged with plain base image name
+        # so that it would be inspectable
```

Comment:
It's inspectable anyway, but is the issue that other plugins are looking for it using the plain image name without a registry? If so, let's make this comment say that.

Reply:
Changed to "other plugins would use base_image to inspect it"